Ajoute un article consacré à ce qui peut être publié sur data.gouv.fr

bboulesteix opened the pull request:
Cette _pull-request_ soumet un premier brouillon d’un article consacré à ce qui peut être publié sur data.gouv.fr et à la meilleure manière de le faire pour encourager la réutilisation des données, comme annoncé dans #36. 

<img width="1440" alt="capture d ecran 2018-11-12 18 35 47" src="https://user-images.githubusercontent.com/3286488/48364890-e6275900-e6a9-11e8-8f4b-3230df56a1ae.png">

Pour rédiger l’article, je me suis appuyé sur les documents suivants :

* [Publishing Open Government Data](https://www.w3.org/TR/gov-data/) (W3C) ;
* [Open Government Data Toolkit](http://opendatatoolkit.worldbank.org/en/index.html) (World Bank) ;
* [A Primer on Machine Readability for Online Documents and Data](https://www.data.gov/developers/blog/primer-machine-readability-online-documents-and-data) (data.gov) ;
* [Open Government Data](https://opengovdata.io) (livre en ligne) ;
* [Open Data Has Little Value If People Can’t Use It](https://hbr.org/2013/03/open-data-has-little-value-if) (Harvard Business Review).

Vos relectures et critiques sont les bienvenues. 👀

bboulesteix committed:
Ajoute un article d’aide didactique

bboulesteix committed:
Corrige un lien

Pour réparer la redirection.

bboulesteix committed:
Corrige le slug d’un article

Le slug de l’article consacré à la publication d’un jeu de données contenait une coquille que ce commit corrige.

bboulesteix committed:
Corrige le slug d’un fichier

abulte commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,131 @@
+---
+# Que publier sur data.gouv.fr et comment le publier
+
+data.gouv.fr est la plateforme ouverte des données publiques française. En ce sens, elle met en relation des producteurs et des utilisateurs de données. Une donnée est considérée comme ouverte quand elle peut être consultée, utilisée, et partagée par tous, sans restriction.
```
Re "sans restriction", il y a quand même des obligations liées aux différentes licences le cas échéant (paternité, réciprocité...).

bboulesteix replied:
Bien vu, je rectifie. Merci. 👍

abulte commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,131 @@
+
+#### Données tabulaires
+
+Les données tabulaires (feuilles de calcul), gagnent à être publiées sous la forme de fichiers CSV (_comma-separated values_), reconnaissables à leur extension en `.csv`. Si le fichier que vous avez entre les mains est un fichier Excel, nous vous conseillons d’exporter vos données au format CSV. [Voir comment exporter des données au format CSV à partir d’Excel](https://support.office.com/fr-fr/ar ...
```
En regardant rapidement (https://stackoverflow.com/questions/4221176/excel-to-csv-with-utf8-encoding) cette méthode ne produit pas de l'UTF-8, ce qui est fortement conseillé... Peut-être faut-il pointer vers une méthode alternative ?

bboulesteix replied:
Oui, j’en ai parlé avec @cquest. J’ai changé le lien présent dans l’article. L’article redirige maintenant vers [un article d’IBM](https://www.ibm.com/support/knowledgecenter/fr/SSWU4L/WebLanding/imc_WebLanding/WebLanding_q_a_watson_assistant/Saving_a_CSV_file_with_UTF-8_encoding.html).

abulte commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,131 @@
+
+### Choisir une licence
+
+Par défaut, les données publiées sur data.gouv.fr tombent sous le coup du [Code des relations entre le public et l'administration](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=E4583EBE2857A0DF9056D5D0082F8F2D.tplgfr31s_2?idSectionTA=LEGISCTA000032255212&cidTexte=LEGITEXT000031366350&dateTexte=20181016), qui dispose notamment que :
```
Même quand les données sont publiées par un organisme non public ? Société civile, individu...

bboulesteix replied:
Bonne question. Je ne sais pas. 🤔

bboulesteix replied:
Question posée à Maud. 🔎 Je laisse la discussion ouverte.

bboulesteix replied:
Réponse de Maud reçue et paragraphe ajouté dans l’article en conséquent. Je ferme donc cette discussion.

abulte commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnees.md:
```diff
@@ -1,6 +1,6 @@
 ---
 title: Publier un jeu de données
-slug: publier-un-jeu-de-donnes
+slug: publier-un-jeu-de-donnees
```
Redirection à ajouter.

bboulesteix replied:
Exact, bien vu. Merci. 👍

abulte reviewed (commented):
Je crois comprendre qu'on a évacué la question des PDF en n'en parlant pas ;-) A la limite ça me va dans un premier temps, tant qu'on a pas une doctrine forte en interne.

bboulesteix committed:
Modifie l’article suite à une discussion avec Christian

Ce commit modifie la version initiale de l’article pour prendre en compte les modifications proposées par Christian.

bboulesteix committed:
Supprime un passage de l’article

bboulesteix committed:
Met en place une redirection

bboulesteix commented:
Pour répondre à ta remarque, @abulte :

> Je crois comprendre qu'on a évacué la question des PDF en n'en parlant pas ;-)

Nous pourrions faire ça, oui, mais nous avons pris une autre décision avec @cquest, à savoir, en parler dans le but d’établir une ligne claire : **non** aux fichiers PDF isolés ; **oui** aux _listes_ de fichiers PDF. Je t’invite à lire [le paragraphe dans lequel nous formulons ce raisonnement](https://github.com/etalab/doc.data.gouv.fr/pull/56/files#diff-999d0d0e0ec046415aaaaaba546bdabdR86) et à me faire signe si tu le juges insuffisant.

abulte commented:
@bboulesteix ok je vais lire le paragraphe. Dans un souci de transparence du débat, j’aurais aimé que votre discussion ait eu lieu ici en revanche ;-)

bboulesteix commented:
> @bboulesteix ok je vais lire le paragraphe. Dans un souci de transparence du débat, j’aurais aimé que votre discussion ait eu lieu ici en revanche ;-)

C’est noté. Je suis d’accord avec cette idée, cette façon de procéder. La discussion que nous eue avec @cquest aurait sans doute pu se tenir ici, dans les commentaires de la _pull-request_, même si je reconnais que le fait d’avoir eu une conversation en face à face nous a sans doute fait gagner du temps — au détriment de l’archivage de la discussion, j’en conviens. 🙂

taniki commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,117 @@
+
+En revanche, une _liste_ de photographies peut être considérée comme un jeu de données, à condition que cette liste prennent la forme d’un CSV, dans lequel chaque ligne contient des informations sur chaque photo : lieu de la prise de vue, date, URL du fichier, auteur du cliché, et ainsi de suite…
+
+#### Documents bureautique
```
```suggestion
#### Documents bureautiques
```

taniki commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,117 @@
+* XML ;
+* YAML.
+
+#### Images et vidéos
```
Est-ce qu'on prendrait pas un peu d'avance et incluerait le "son" ? (:

bboulesteix replied:
Pourquoi pas. Bonne idée, merci. 👍

taniki committed:
Modifie que-publier-et-comment-le-publier.md

Corrige une coquille.

Co-Authored-By: bboulesteix <brice.boulesteix@data.gouv.fr>

Commit:
Modifie que-publier-et-comment-le-publier.md

Ajoute les sons dans la section `Images et vidéos`.

abulte commented on articles/_a-propos/que-publier-et-comment-le-publier.md:
```diff
@@ -0,0 +1,117 @@
+
+#### Documents bureautiques
+
+Même raisonnement que pour les images et les vidéos : les fichiers Word, PowerPoint, ou PDF, ne constituent pas nécessairement des jeux de données, par conséquent leur importation sur data.gouv.fr est déconseillée. Cela dit, une liste de fichiers PDF peut constituer un jeu de données, si la liste prend la forme d’un CSV qui contient des informations supplémentaires sur chaque PDF.
```
J’aime bien l’idée de considérer qu’un corpus est de la donnée 👍 Mais du coup je ne vois pas pourquoi on ne pourrait pas uploader un corpus de PDF directement.

bboulesteix replied:
Rien, dans le paragraphe, n’interdit d’importer un lot de PDF sur data.gouv.fr, me semble-t-il. Mais je me trompe peut-être — un peu l’impression d’être juge et partie ici. 😎 

Ce passage vise à tracer une ligne entre l’importations de documents désordonnés, dépourvus de cohérence, et l’importation d’un _corpus_ de documents, reliés entre eux par un fichier CSV.

abulte replied:
Ce n’est pas clair pour moi. Je vais proposer une autre version si tu veux bien.

bboulesteix replied:
Ça me va. 🙂 Merci à toi.

bboulesteix replied:
As-tu eu le temps de préparer un amendement à mon texte @abulte ? Je fais référence à la discussion ci-dessus. ☝️

abulte replied:
Oui ;-) Désolé pour le délai.

> Cela dit, une liste de fichiers PDF peut constituer un jeu de données, si la liste *est accompagnée* d’un CSV qui contient des informations supplémentaires sur chaque PDF.

bboulesteix replied:
Merci pour ta proposition, @abulte. Je vais l’intégrer dans le texte. 👍

Commit:
Modifie que-publier-et-comment-le-publier.md

Pour intégrer la proposition d’Alexandre.

Commit:
Change le style de que-publier-et-comment-le-publier.md

Modifie la syntaxe pour harmoniser la présentation des fichiers Markdown.

Commit:
Cas des particuliers qui publient sans licence

Ce _commit_ ajoute un paragraphe consacré au cas des particuliers qui publient des données sans choisir une licence. Il doit beaucoup à Maud.

Commit:
Modifie que-publier-et-comment-le-publier.md

Met des tirets courts à la place des tirets moyens.

Commit:
Corrige une coquille

bboulesteix commented:
Je pense que la _pull request_ est mûre pour être relue. 👀 En sachant que le cas particulier du menu déroulant qui permet d’associer une licence à un jeu de données est évoqué plus en détails dans : https://github.com/etalab/data.gouv.fr/issues/114.
